jsonnet/kube-prometheus/prometheus/prometheus.libsonnet

```diff
@@ -178,7 +178,7 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
         spec: {
           replicas: p.replicas,
           version: $._config.versions.prometheus,
-          image: $._config.imageRepos.prometheus + ':' + $._config.versions.prometheus,
+          image: $._config.imageRepos.prometheus,
```

Comment:
`image` should be the full image as it was before, if that doesn't work, then we need to fix the prometheus operator in that regard, I think it was the combination of specifying `version`, `tag`, and the full `image` that caused the edge case to trigger before

Reply:
Full image with tag in the image does not work, as the version gets added to it, hence the initial failure, after removing that and adding either tag or leaving version alone that fixes it. We did not have tag before, just image and version.

Reply:
Then this is broken and needs to be fixed in prometheus operator

Reply:
No, sorry for not being clear enough it was just full `image` and `version`. I reverted the changes to rerun the CI again.

Reply:
Yep fails, so we fix it in master and latest release before we can bump this?

Reply:
yes